src/pages/Troubleshooting.vue

```diff
@@ -71,18 +70,6 @@ export default {
     state:    ipcRenderer.sendSync('k8s-state'),
     settings: defaultSettings,
   }),
-  computed:   {
-    canFactoryReset() {
-      switch (this.state) {
-      case K8s.State.STOPPED:
-      case K8s.State.STARTED:
-      case K8s.State.ERROR:
-        return true;
-      default:
```

Comment:
This was returning `false` if we're `STARTING` or `STOPPING`.

I can't find any actual problems with this, though, going through the code.

Reply:
Yeah, it didn't allow a reset if the system was stuck `STARTING`. I don't think it happens as often when the system is `STOPPING`, but it's less likely that a user will want to do a factory-reset when it's shutting down.